cmd/proxy/actions/app.go

```diff
@@ -41,7 +41,11 @@ func App(conf *config.Config) (*buffalo.App, error) {
 
 	// mount .netrc to home dir
 	// to have access to private repos.
-	initializeNETRC(conf.Proxy.NETRCPath)
+	initializeDotFile(conf.Proxy.NETRCPath)
```

Comment:
dotFiles can be many things (.env, .gitignore, etc). So let's call this something more clear of the task: moving an auth file to the root directory. So something like: `initializeAuthFile`.

Reply:
Should I put this function in cmd/proxy/actions/auth.go?

Reply:
or init.go, both are ok i guess